site/content/entry/accurate-audio-emulation.md

```diff
@@ -0,0 +1,199 @@
+So for emulation, where the goal of many developers is not only to make the emulation accurate, but also to make it *fast*, HLE audio is a great middle ground as you get to have high accuracy while also taking almost no processing effort.
+But as usual, there is one thing thats particularly hard to get right with HLE audio.
+In order to write an effective HLE audio, one must first reverse engineer the audio code that the game uses, and truly understand what it does.
+When working on HLE audio, merry spent a long time writing tools to help break down exactly what the game was doing, and also decipher what the audio code is doing *semantically*.
+This turns out to be pretty tricky in practice.
+It's much simpler to look at disassembly and see what its doing than it is to really understand why its doing something, which is a requirement when recreating the audio code.
+Simply put, writing HLE audio support means diving deep into how the code for the game's audio works, and recreating its functionality in Citra, without ever running any actual audio code from the game.
+But there's a very different way to handle audio, and this is hinted about at the end of the 2016 article: LLE Audio Emulation.
+
+{did I mention it also fixes fire emblem whatever audio?}
+
+## Debugging Pokémon X Told Us That It's Broken but not WHY
+
+Before looking in depth at what LLE audio emulation is all about, a quick diversion into the debugging effort that went into Pokémon X / Y is in order.
+Looking at the [Citra YouTube channel](https://www.youtube.com/channel/UC_dcdgzuapBtAY4ol3x-90Q), one will find several videos talking about the games progress!
+At the time, we were just as excited as everyone else to see how well the games were advancing, but at some point, they stopped getting better.
+After CRO support landed to get the games running (thanks to both [Subv](https://github.com/Subv/) and [wwylele](https://github.com/wwylele/)), several fixes to the 3DS GPU emulation, followed by geometry shader support (thanks to the hard work of [neobrain](https://github.com/neobrain/), [JayFoxRox](https://github.com/JayFoxRox/), [ds84182](https://github.com/ds84182), and once again [wwylele](https://github.com/wwylele/)), we really hoped that the games would finally start working!
+As everyone knows, they still didn't work!
+We kept feeding more and more time into features that made the emulation so much better, yet this one very popular, and very stubborn game would not work.
+[Subv](https://github.com/Subv/) spent many long hours reverse engineering the game, and found that at the core of the game lay a state machine that drives the game engine.
+The game would transition from state to state, and mysteriously whenever the game softlocked, it simply wasn't moving onto the next state.
+As cool as it is to what causes the softlock, it doesn't answer the big question of *why* doesn't the game transition to the next state like it should.
```

Comment:
Should it be "*why* the game doesn't transition to the next state" instead of 'doesn't the game'